package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "compendium-cdk",
+  "description": "Compendium for various stuff built on serverless stack",
+  "version": "0.0.2",
+  "module": "lib/index.js",
+  "main": "lib/index.js",
+  "author": {
+    "name": "Jakub Knejzlik",
+    "email": "jakub.knejzlik@gmail.com"
+  },
+  "dependencies": {
+    "@aws-cdk/aws-appsync-alpha": "^2.55.1-alpha.0",
+    "@aws-sdk/types": "^3.127.0",
+    "@tsconfig/node16": "^1.0.1",
+    "aws-cdk": "^2.39.0",
+    "aws-cdk-lib": "2.59.0",
+    "constructs": "^10.0.0",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.8.4"
```

Comment:
Why do we have typescript as main dependency as well as devDependency? Also let's use same ts version across whole project, we are using multiple versions (4.6.3, 4.8.4, 4.9.4)

Reply:
aligned in parent package.json